dev/components/conversationsList/tpl.pug

```diff
@@ -10,7 +10,7 @@
             span(ng-if="!vm.toUser && (!item.email || item.email === null || item.email === undefined)") Public
             span(ng-if="vm.toUser") from: {{item.author}}
           .message
-            .bubble(ng-if="vm.toUser && !item.CONVERSATION_IS_VIEWED")
+            .bubble(ng-if="vm.unreadMessage && item.message")
 
```

Comment:
Are you sure that it will not show messages thet user send to sombody?

Reply:
I've tested

Reply:
Tested again.
And there's different problem.